docs/user/general/values.rst

```diff
@@ -46,6 +46,21 @@ Here is an example::
     | Dale        | null                      | True                |
     +-------------+---------------------------+---------------------+
 
+
+NULL Literal Handling
+---------------------
+
+Because the type of a null literal is unknown, a special data type ``UNDEFINED`` is reserved to allow it be accepted as a valid function argument::
+
+    od> SELECT NULL, NULL = NULL, 1 + NULL, LENGTH(NULL);
+    fetched rows / total rows = 1/1
+    +--------+---------------+------------+----------------+
+    | NULL   | NULL = NULL   | 1 + NULL   | LENGTH(NULL)   |
```

Comment:
1 + NULL should 1 or NULL?
LENGTH(NULL) should be NULL?

Reply:
yes, it should be what is documented here. Line 34 explains this. Unless the operator/function is special, expression involving NULL will end with NULL.